app/webpacker/styles/layout.scss

```
@@ -77,7 +77,7 @@ main.semantic {
     margin: 1em auto 4em;
   }
 
-  > article {
+  > article:not(.fullwidth) {
```

I think explicitly stating how wide a full width article should be might be clearer here

```suggestion
  > article {
    &.fullwidth {
      flex: 1 0 100%;
    }
  }
```